src/clients/inst_autoconfigure.rb

```diff
@@ -87,7 +87,7 @@ def main
         log.error "Could not process these unknown profile sections: #{unknown_sections}"
         needed_packages = Y2Autoinstallation::PackagerSearcher.new(
           unknown_sections
-        ).evaluate_via_schema
+        ).evaluate_via_rpm
```

Comment:
well, how it works when system is not registered or if some repository is not available?

Reply:
In that case the user gets the info "No needed package found." for this section, which is true. :-)
Anyway normally YAST packages should be  available.....